build.gradle

```diff
@@ -101,6 +100,10 @@ allprojects {
             url "https://maven.research.rackspacecloud.com/content/groups/aggregate/"
         }
     }
+
+    ext {
+        docsHome = "http://www.openrepose.org/versions/$version/index.html".toString()
+    }
```

Comment:
What is the difference between putting the external property on every project versus putting it on the root project? Out of curiosity.